wlcreate/Start-Asian-Love
# Resolves 2. As a user, I want to see general information of the increase in hate crimes on the Home page

## Description



This PR resolves issue number: #2 

This PR adds `react-player` as a dependency and text that describes the current situation and what Start Asian Love is. I decided to use `react-player` to embed the playlist after doing some research and followed along with the documentation to make the video responsive. 

Resources:
- [How to embed](https://coderrocketfuel.com/article/embed-a-youtube-video-in-a-react-webpage)
- [React Player Repo](https://github.com/CookPete/react-player)
- [React Player Demo Source Code](https://github.com/CookPete/react-player/blob/master/src/demo/App.js)

## Acceptance Criteria

- Includes a Youtube playlist of videos with a main video for the user to watch directly
- Written content
    - Short intro of us & why we are writing this
    - Summary of what's been happening
    - Optional: Call to action for browsing the site (support, wins, etc.)

## Self-check

Please check all that apply:



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓  | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

<img width="1505" alt="Screen Shot 2021-04-08 at 8 11 01 PM" src="https://user-images.githubusercontent.com/62153993/114110878-8cd9a900-98a6-11eb-8c05-8bf0ed7dbd88.png">

### After

[![Image from Gyazo](https://i.gyazo.com/2f9736641956bb126a65a92114de6a0e.gif)](https://gyazo.com/2f9736641956bb126a65a92114de6a0e)

## Testing Steps / QA Criteria


- switch to this feature branch: `git checkout general-information-home`
- install the dependencies: `npm install`
- start the app: `npm run dev`
- open the browser and go to the home page to see if there is a YouTube playlist to interact with and text
